src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/IChatClient.cs

```diff
@@ -26,27 +26,35 @@ namespace Microsoft.Extensions.AI;
-    /// <param name="options">The chat options to configure the request.</param>
+    /// <param name="chatMessages">The list of chat messages to send and to be augmented with generated messages.</param>
+    /// <param name="options">The chat options with which to configure the request.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>The response messages generated by the client.</returns>
+    /// <exception cref="ArgumentNullException"><paramref name="chatMessages"/> is <see langword="null"/>.</exception>
     /// <remarks>
-    /// The returned messages aren't added to <paramref name="chatMessages"/>. However, any intermediate messages generated implicitly
-    /// by the client, including any messages for roundtrips to the model as part of the implementation of this request, are included.
+    /// The response message generated by <see cref="GetResponseAsync"/> is both returned from the method as well as automatically
+    /// added into <paramref name="chatMessages"/>. Any intermediate messages generated implicitly as part of the interaction are
+    /// also added to the chat history. For example, if as part of satisfying this request, the <see cref="GetResponseAsync"/> method
+    /// itself issues multiple requests to one or more underlying <see cref="IChatClient"/> instances, all of those messages will also
+    /// be included in <paramref name="chatMessages"/>.
     /// </remarks>
     Task<ChatResponse> GetResponseAsync(
         IList<ChatMessage> chatMessages,
         ChatOptions? options = null,
         CancellationToken cancellationToken = default);
 
     /// <summary>Sends chat messages and streams the response.</summary>
-    /// <param name="chatMessages">The chat content to send.</param>
-    /// <param name="options">The chat options to configure the request.</param>
+    /// <param name="chatMessages">The list of chat messages to send and to be augmented with generated messages.</param>
+    /// <param name="options">The chat options with which to configure the request.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>The response messages generated by the client.</returns>
+    /// <exception cref="ArgumentNullException"><paramref name="chatMessages"/> is <see langword="null"/>.</exception>
     /// <remarks>
-    /// The returned messages aren't added to <paramref name="chatMessages"/>. However, any intermediate messages generated implicitly
-    /// by the client, including any messages for roundtrips to the model as part of the implementation of this request, are included.
+    /// The response updates generated by <see cref="GetStreamingResponseAsync"/> are both stream from the method as well as automatically
```

Comment:
```suggestion
    /// The response updates generated by <see cref="GetStreamingResponseAsync"/> are streamed from the method as well as automatically
```

Or "returned" instead of "streamed" would do.

I recommend we avoid the word "both" in this context since in one interpretation it would suggest there are going to be exactly two response updates.